tiup/tiup-component-dm-prune.md

```diff
@@ -0,0 +1,25 @@
+---
+title: tiup dm prune
+---
+
+# tiup dm prune
+
+When scaling in the cluster<!--(/tiup/tiup-component-dm-scale-in.md) -->, a small amount of metadata in etcd is not cleaned up, which usually causes no problem. If you need to clean up the metadata, you can manually execute the `tiup dm prune` command.
```

Comment:
```suggestion
When you scale in the cluster<!--(/tiup/tiup-component-dm-scale-in.md) -->, a small amount of metadata in etcd is not cleaned up, which usually causes no problem. If you need to clean up the metadata, you can manually execute the `tiup dm prune` command.
```
Note that when the subject in the subordinate clause is different from that of the main clause, you can't omit the former.